examples/e2e/app-router/package.json

```diff
@@ -10,8 +10,9 @@
     "lint": "next lint",
     "clean": "rm -rf .turbo node_modules .next .open-next",
     "build:worker": "pnpm opennextjs-cloudflare",
-    "dev:worker": "wrangler dev --port 8770 --inspector-port 9330",
-    "preview": "pnpm build:worker && pnpm dev:worker"
+    "dev:worker": "wrangler dev --port 8790 --inspector-port 9350",
+    "preview": "pnpm build:worker && pnpm dev:worker",
+    "e2e-fix": "playwright test -c e2e/playwright.config.ts"
```

Comment:
`preview` doesn't seem to work as this app is missing the wrangler dependency, consecutively `e2e-fix` also doesn't work

unless I'm doing something wrong?
![Screenshot 2025-01-28 at 14 51 14](https://github.com/user-attachments/assets/39d83eff-7f51-49a1-83ae-4d98e5a8cfa0)

Reply:
Good point, I missed adding wrangler in all imported apps probably, I'll amend the PRs.
It works locally for me, probably wrangler is installed in a parent folder